### ISSUE TITLE
Output translation in text 

### ISSUE THREAD

**Issue by wetfox** [ppisljar/pdf_translator#7]
Thank you for this project

Is it possible to have the translated text to be output as text and not images in the PDF?

**Comment by ppisljar** [comment]
yes, this is on my todo list for the future, just not sure when i'll get to it.
